src/components/Landing/Landing.tsx

```diff
@@ -0,0 +1,121 @@
+import { useState } from "react";
+import {
+  Box,
+  Button,
+  Typography,
+  Card,
+  CardMedia,
+  CardContent,
+  Grid
+} from "@mui/material";
+
+import {
+  Login as LoginIcon,
+  StarHalf as StarHalfIcon,
+  Restaurant as RestaurantIcon,
+  GetApp as GetAppIcon,
+  Share as ShareIcon,
+} from "@mui/icons-material";
+import Login from "../Login/Login";
+
+const LandingPage = () => {
+
+  const [open, setOpen] = useState(false);
```

Comment:
Boolean values usually start with `is`, `can`, `has`, etc. They Indicate the possibility of two states: true or false

```suggestion
  const [isOpen, setIsOpen] = useState(false);
```

Reply:
@Cema2019 Could you please update this section of the code? Let’s ensure consistency in variable naming conventions. As @Ralexs0096 suggested, using these prefixes is a standard practice for boolean variables, so let's align this part accordingly.

Reply:
Done!